index.html

```diff
@@ -30,6 +30,126 @@ <h1>Hey there. I'm John Buhendwa <span>I'm a software developer</span></h1>
                 <li><a href="#"><i class="fa fa-angellist"></i></a></li>
             </ul>
         </section>
+        <div class="button-section">
+            <button><i class="fa fa-angle-down" id="next-section"></i></button>
+        </div>
+        <section class="work-section">
+            <h1>My Recent Works</h1>
+            <div id="hr-line"></div>
+            <div class="project-container">
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
```

Comment:
Great job putting your project together :100: There are just a few changes you need to work on.
- [ ] Your  **See Project**  buttons don't seem to follow the right button styles when on hover. Please do well to fix this so your design can be a replica of the Figma design and also meet all the project requirements :wink: 

- Your design when on hover
![image](https://user-images.githubusercontent.com/102699958/212201643-71c71987-9a82-47a8-9408-a60d3fa98158.png)
- The Figma design on hover
![image](https://user-images.githubusercontent.com/102699958/212201741-730fd203-7612-4e97-a14c-1cf01bcc637b.png)